Week3/script.js

```diff
@@ -0,0 +1,278 @@
+// 2.9 Now if unlike Jim, you don't like 'meerkat' and you want to delete it from the array,
+// but you don't know the position or the index of the item in the array, how can you find it?
+console.log(`Using the method indexOf() of the array will help to find the index 'meerkat'`);
+
+// 2.10 Log the index of 'meerkat'. Add a message so it says: 'The item you are looking for is at index: '
+// (here you should show the index of the item).
+console.log(`The item you are looking for is at index: ` + favoriteAnimals.indexOf('meerkat'));
+
+console.log('.............................................');
+
+// More JavaScript
+// 1. Create a function that takes 3 arguments and returns the sum of the these arguments.
+function sumEntries(arg1, arg2, arg3) {
+  return arg1 + arg2 + arg3;
+}
+
+console.log(`sumEntries(5, 6, 9)`);
+console.log(`The result is: ` + sumEntries(5, 6, 9));
+
+// 2. Create a function named colorCar that receives a color, and prints out, 'a red car' for example.
+function colorCar(color) {
+  console.log(`a ` + color + ` car`);
+}
+
+console.log(colorCar(`red`));
+console.log(colorCar(`blue`));
+
+// 3. Create an object and a function that takes the object as a parameter
+// and prints out all of its properties and values.
+const obj = {
+  color: `blue`,
+  type: 2,
+};
+
+function printObj(obj) {
+  console.log(obj);
+}
+
+printObj(obj);
+
+// 4. Create a function named vehicleType that receives a color, and a code, 1 for car, 2 for motorbike.
+// And prints 'a blue motorbike' for example when called as vehicleType("blue", 2)
+function vehicleType(color, code) {
+  if (code === 1) {
+    console.log(`a ` + color + ` car`);
+  } else if (code === 2) {
+    console.log(`a ` + color + ` motorbike`);
+  } else {
+    console.log(`Please, insert code 1 for 'car' or 2 for 'motorbike'`);
+  }
+}
+
+vehicleType(`yellow`, 1);
+vehicleType(`green`, 2);
+vehicleType(`blue`, 3);
+
+// 5. Can you write the following without the if statement,
+// but with just as a single line with console.log(...);?
+console.log(`The answer is: Yes, I can write it as a single line`);
+3 === 3 ? console.log(`yes`) : console.log(`no`);
+// or
+console.log(3 === 3 ? `yes` : `no`);
+// Another example:
+3 === 7 ? console.log(`yes`) : console.log(`no`);
+console.log(3 === 7 ? `yes` : `no`);
+
+// 6. Create a function called vehicle, like before, but takes another parameter called age,
+// so that vehicle("blue", 1, 5) prints 'a blue used car'
+function vehicle(color, code, age) {
+  if (code === 1 && age === 3) {
+    console.log(`a ` + color + ` new car`);
+  } else if (code === 2 && age === 3) {
+    console.log(`a ` + color + ` new motorbike`);
+  } else if (code === 1 && age === 5) {
+    console.log(`a ` + color + ` used car`);
+  } else if (code === 2 && age === 5) {
+    console.log(`a ` + color + ` used motorbike`);
+  } else {
+    console.log(`Please, insert code 1 for 'car' or 2 for 'motorbike'
+      Please, insert age 3 for 'new' or 5 for 'used'`);
```

Comment:
This works but it would be nice if a car could have an age of 4, can you change the function so that cars that have an age of 0 or 1 are new and anything else is marked as used?